### PR TITLE
Fe update scan page API

### PR DESCRIPTION
**Pull request by lcampbell2**
update Scan details dage to js-api
Screenshot from 2021-01-26 09-02-40
Screenshot from 2021-01-26 09-02-48

### CODE REVIEW

**Review by tparrott-cse — COMMENTED** [review]
In the first image you have web scan with email results...?

**Review by sleepycat — APPROVED** [review]
:tada:  Nice!

File: frontend/src/graphql/queries.js [hunk 1]
```diff
@@ -174,6 +174,162 @@ export const GET_GUIDANCE_TAGS_OF_DOMAIN = gql`
...
+        }
+      }
+      email {
+        dkim(first: 10) {
+          edges {
+            cursor
+            node {
+              id
+              timestamp
+              results {
```
Comment (FestiveKyle) [inline frontend/src/graphql/queries.js:245]: results need pagination arguments

**Review by FestiveKyle — CHANGES_REQUESTED** [review]

**Comment by lcampbell2** [comment]
> In the first image you have web scan with email results...?

Faker only provided tags for dkim, I'll update the pics with real results

**Review by FestiveKyle — APPROVED** [review]
Great stuff 🔥

**Pull request merged**